blueprints/app/files/.eslintrc.js

```diff
@@ -57,10 +57,32 @@ module.exports = {
         node: true,
       },
       extends: ['plugin:n/recommended'],
-    },
+    },<% if (templateTag) { %>
+    // Template tag files
+    {
```

Comment:
to do this we need to move everything out of the root-object of the .eslintconfig and move everything to overrides because we can't have top-level parsers: https://github.com/ember-cli/ember-cli/pull/10447/files#diff-042c163c37338253d4a1cc6bf038fb8b4b45eebef94ebbd439a81c38b158dcb6R5